websites.yml

```diff
@@ -46,3 +46,21 @@ username_patterns:
     invalid_patterns:
       - "\\.(com|net)"
       - "(\\.)\\1{1,}"  # consecutive '.' not allowed
+avatar:
+  pinterest:
+    property: image_xlarge_url
+  gitlab:
+    property: og:image
```

Comment:
This isnt a property.  This is a standard.  It is *the* standard.  If you dont like `gitlab: true` to reflect using OpenGraph, then please use

Reply:
@jayvdb : Not sure if I understand what you mean correctly but this is for future compatibility, if there is a website added in future which has `twitter:image`( see [this](https://developer.twitter.com/en/docs/tweets/optimize-with-cards/overview/summary) ) but not `og:image`, all that needs to be done is just write `twitter:image` here(instead of `og:image`) for it to work.
This also allows for supporting arbitrary meta tags for the image.

Reply:
Maybe you mean this should be renamed to `meta-name` or something?
Or that this is an unnecessary feature? Let me know if it's the second case, I'll remove it.

Reply:
What I want is `gitlab: true`.  OpenGraph is the default.  Nothing else is needed.  Any other alternative is an override.  And that can be designed later.

Reply:
Ok, so I'll shift this to
```
gitlab:
  opengraph: true
```
so we can still have url overrides easily.

Reply:
`gitlab: true` just coexists fine with
```
facebook:
  url: ...
```
This is YAML.
You are using YAML to store data in a JSON-like structure.  That is not YAML.